src/main/java/mdlaf/utils/MaterialDrawingUtils.java

```diff
@@ -46,22 +47,23 @@
     public static Graphics getAliasedGraphics(Graphics g) {
         Map<RenderingHints.Key, Object> hints = (Map<RenderingHints.Key, Object>) Toolkit.getDefaultToolkit().getDesktopProperty("awt.font.desktophints");
 
-        if (hints != null) {
-            hints.put(RenderingHints.KEY_ANTIALIASING, RenderingHints.VALUE_ANTIALIAS_ON);
-            hints.put(RenderingHints.KEY_RENDERING, RenderingHints.VALUE_RENDER_DEFAULT);
-            hints.put(RenderingHints.KEY_COLOR_RENDERING, RenderingHints.VALUE_COLOR_RENDER_DEFAULT);
-            //hints.put(RenderingHints.KEY_TEXT_ANTIALIASING, RenderingHints.VALUE_TEXT_ANTIALIAS_LCD_HRGB);
-            hints.put(RenderingHints.KEY_FRACTIONALMETRICS, RenderingHints.VALUE_FRACTIONALMETRICS_ON);
-            hints.put(RenderingHints.KEY_TEXT_ANTIALIASING, RenderingHints.VALUE_TEXT_ANTIALIAS_ON);
-            //hints.put(RenderingHints.KEY_TEXT_ANTIALIASING,	RenderingHints.VALUE_TEXT_ANTIALIAS_GASP);
-
-            Graphics2D g2d = (Graphics2D) g;
-            g2d.addRenderingHints(hints);
-            return g2d;
+        if (hints == null) {
+            hints = new HashMap<>();
```

Comment:
I'm not completely happy with this solution, because could have happened a collision between propriety.

But this is a problem that I want to solve and I'm happy to work around with you to choose the best solution to use in this case.